rubygem-bootstrap-sass/rubygem-bootstrap-sass.spec

```diff
@@ -5,8 +5,8 @@
 
 Summary: Twitter's Bootstrap, converted to Sass and ready to drop into Rails or Compass
```

Comment:
This changed to `bootstrap-sass is a Sass-powered version of Bootstrap 3, ready to drop right into your Sass powered applications.` (though it'll need trimming to 72 chars).